nltk/translate/api.py

```diff
@@ -163,51 +157,6 @@ def invert(self):
         return AlignedSent(self._mots, self._words,
                                self._alignment.invert())
 
-    def precision(self, reference):
-        """
-        Return the precision of an aligned sentence with respect to a
-        "gold standard" reference ``AlignedSent``.
-
-        :type reference: AlignedSent or Alignment
-        :param reference: A "gold standard" reference aligned sentence.
-        :rtype: float or None
-        """
-        # Get alignments in set of 2-tuples form
-        # The "possible" precision is used since it doesn't penalize for finding
-        # an alignment that was marked as "possible" (NAACL corpus)
-
-        align = self.alignment
-        if isinstance(reference, AlignedSent):
-            possible = reference.alignment
-        else:
-            possible = Alignment(reference)
-
-        return precision(possible, align)
-
-
-    def recall(self, reference):
```

Comment:
Since `recall()` is removed, the doctests in `test\translate.doctest` will have to be updated.